cspell.json

```diff
@@ -355,6 +355,7 @@
         "src/library/Registrar/**",
         "src/library/Server/Manager/**",
         "src/library/Box/**",
+        "src/library/FOSSBilling/**",
```

Comment:
Why are we adding this to the exclusions?
The only reason why the other's are excluded is because they had 100s of failures

Reply:
There was some issues with the moved `Box` library classes, likely part of the reason the Box folder was excluded. 

I felt that rather than adding a bunch of words to the exclusion list, the spelling issues should be resolved as part of an independent PR specific to that issue. However, I'm happy to review this and see about removing this exclusion.

Reply:
As long as it actually gets done I'm okay with it. Although perhaps we can exclude just the moved classes rather than all of them? I'd just prefer to know that we actually are checking and ensuring spelling is correct in our own classes going forward

Reply:
It wasn't as bad as I thought 🙂 - I've corrected a minor spelling issue and just excluded certain strings, so now we can include the FOSSBilling folder.